test/e2e/webhelp.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { expect, test } from '@playwright/test';
+import { testSelector } from '../utils';
+import { WebHelpPage } from '../page/webhelp-page';
+import {  RESOLUTIONS } from '../visual/visual-constants';
```

Comment:
`  ` suggest me file doesn't linted. 😔

Reply:
the files in test/ are not linted indeed
I suggest to fix it in another pr